test/Entity/AccessTokenEntityTest.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-expressive-authentication-oauth2 for the canonical source repository
+ * @copyright Copyright (c) 2017 Zend Technologies USA Inc. (https://www.zend.com)
+ * @license   https://github.com/zendframework/zend-expressive-authentication-oauth2/blob/master/LICENSE.md
+ *     New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace ZendTest\Expressive\Authentication\OAuth2\Entity;
+
+use League\OAuth2\Server\Entities\AccessTokenEntityInterface;
+use PHPUnit\Framework\TestCase;
+use Zend\Expressive\Authentication\OAuth2\Entity\AccessTokenEntity;
+
+class AccessTokenEntityTest extends TestCase
```

Comment:
I don't think we need this test. If it's just testing that the constructor returns an instance of the class, that's something we know already.

If there's other behavior to test — e.g., default property values, property values based on constructor arguments, etc. — then tests make sense.

Reply:
If I remove the AccessTokenEntityTest we need to use `@codeCoverageIgnore` to omit this class from the code coverage. IMHO I think a test class like this, even if just checking for constructor, can be useful as a constrain for future changes. For instance, if someone changes the `AccessTokenEntityInterface` this test is able to intercept it.